src/ui/components/panels/loadout/active_loadout/components/_PilotAttack.vue

```diff
@@ -683,6 +687,9 @@ export default Vue.extend({
       this.kill = false
       this.confirmed = false
       this.overkillHeat = 0
+      this.$nextTick(function () {
+        this.resetAttackRoll = true
```

Comment:
This is a weirdness, but is necessary so that if someone accidentally clicks on skirmish and closes out that it will roll again when clicked again.